pkgs/ok_http/lib/src/ok_http_client.dart

```diff
@@ -21,6 +21,49 @@ import 'package:jni/jni.dart';
 
 import 'jni/bindings.dart' as bindings;
 
+/// Configurations for the [OkHttpClient].
+class OkHttpClientConfiguration {
+  /// The maximum duration (in milliseconds) to wait for a call to complete.
+  ///
+  /// If a call does not finish within the specified time, it will throw a
+  /// `ClientException` with the message "java.io.InterruptedIOException...".
+  /// Default is set to `0`, which indicates no timeout.
+  ///
+  /// See [OkHttpClient.Builder.callTimeout](https://square.github.io/okhttp/5.x/okhttp/okhttp3/-ok-http-client/-builder/call-timeout.html).
+  final int callTimeout;
```

Comment:
Should these be `Duration`?

Reply:
Yeah, `Duration` should be better. Done!